examples/webgl_buffergeometry_instancing_lambert.html

```diff
@@ -292,15 +285,16 @@
 
 			} );
 
+			material.onBeforeCompile = function( shader ) {
+				shader.vertexShader = customLambertVertexShader;
+			};
 			material.defines = material.defines || {};
 			material.defines[ 'INSTANCED' ] = "";
 
 
 			// custom depth material - required for instanced shadows
 
-			var shader = THREE.ShaderLib[ 'customDepthRGBA' ];
-
-			var uniforms = THREE.UniformsUtils.clone( shader.uniforms );
+			var uniforms = THREE.UniformsUtils.clone( customDepthToRGBAShader.uniforms );
 
 			var customDepthMaterial = new THREE.ShaderMaterial( {
```

Comment:
Custom shader can also be replaced with `THREE.MeshDepthMaterial` and explicitly modified with `onBeforeCompile` like the other suggestion.

Reply:
Done!

Reply:
So, now, what is the point of `#ifdef INSTANCED`?

Reply:
True, it's not really needed anymore, removed it.

Reply:
Do you think it is still easy to see the changes required to support instancing?

Reply:
I think so. I would just add the missing `//instanced` on `LambertVertexShader` and it should still be clear what are the changes necessary.

Reply:
Added the // instanced comment.